netctl/netctl.go

```diff
@@ -401,3 +402,14 @@ func dumpList(ctx *cli.Context, list []map[string]interface{}) {
 	os.Stdout.Write(content)
 	os.Stdout.WriteString("\n")
 }
+
+func showVersion(ctx *cli.Context) {
```

Comment:
We could discuss the goals of versioning binaries, APIs, object-model in more detail and separately. And I am completely ok if we wish to go ahead and merge this as well.

But just for completeness, let me summarize how I see the features current patch is trying to bring in; value they add and potential missing things that we are all agreeing to fix somehow in future. Please correct me if I am missing something here:
- bring in per binary `--version`
  - this is immediately useful as right now we don't have a way to know versions for our binaries once they are downloaded and unzipped from github release.
- bring in an extra REST endpoint in netmaster for getting it's binaries version
  - this possibly brings in a value to dump netmaster's version from single place
  - but it is not complete, in many ways:
    - it doesn't address something similar for netplugin and k8s plugins.
    - it doesn't deal with multiple instances of netmaster itself
    - it is not useful for compatability as well since it doesn't talks about what API versions are being used by the client and server
    - we are adding a new endpoint now (ironically for bringing in versions), without addressing what happens when we decide to change contents of `version.Info` structure itself.